components/Stats.vue

```diff
@@ -86,6 +160,79 @@ export default {
         padding: 4px 8px;
       }
     }
+    .county-select-container {
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      align-items: center;
+      label {
+        font-weight: bold;
+        font-size: 30px;
+        color: black;
+        margin-right: 10px;
+      }
+      .county-stats {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        text-align: center;
+        .stat-number {
+          padding: 10px;
+          margin: 20px 0px;
+          font-size: 60px;
+          color: black;
+          font-weight: bold;
+        }
+        .border {
+          border-right: 2px solid lightgray;
+        }
+      }
+      .county-select {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+    }
+  }
+
+  .select-css {
+    display: block;
+    font-size: 16px;
+    font-family: sans-serif;
```

Comment:
Is there any reason to specify font-family here? We have font-family css in global.scss to make sure to apply for all.